include/seqan/index/index_fm_rank_dictionary_levels.h

```diff
@@ -761,10 +1453,17 @@ length(RankDictionary<TValue, Levels<TSpec, TConfig> > const & dict)
 
 template <typename TValue, typename TSpec, typename TConfig, typename TSize, typename TExpand>
 inline typename Size<RankDictionary<TValue, Levels<TSpec, TConfig> > >::Type
-reserve(RankDictionary<TValue, Levels<TSpec, TConfig> > & dict, TSize newCapacity, Tag<TExpand> const tag)
+reserve(RankDictionary<TValue, Levels<TSpec, TConfig> > & dict, TSize const newCapacity, Tag<TExpand> const tag)
```

Comment:
We are moving more to the convention of "use const whenever possible" without exceptions. It may be pedantic, but it increases readability and consistency.